### PR TITLE
Add troubleshooting tip for outages on mock servers

### PR DESCRIPTION
**Pull request by yiantseng**
Described the response in case of outages and added a link to status page

### CODE REVIEW

**Comment by yiantseng** [comment]
Customer reached out about what error codes are provided in case of an outage, I added a quick blurb on it to the troubleshooting tips section.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 1]
```diff
@@ -313,6 +313,8 @@ You can use the mock call log to troubleshoot your requests to mock servers.
 
 If you see `No matching requests` listed in the __Response__ column, this may mean that your mock server is not setup correctly. Make sure [you have an example saved for the request](/docs/postman/mock-servers/mocking-with-examples) in the collection you have the mock connected to.
 
+In the case of a service outage, you will get a 502/503/504 response. Please subscribe and check our [status page](https://status.postman.com/) for updates if you encounter this.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:316]: could we change "our" to "the Postman" - we avoid first person language in docs..

Comment (yiantseng) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: sounds good, just updated it!

**Comment by SueSmith** [comment]
@yiantseng we'll be publishing updates on Mon for the app release so this will go to the prod site then too 🙌

**Pull request merged**